src/www/system_advanced_admin.php

```diff
@@ -636,6 +714,16 @@ function reloadWaitOld () {
                   </div>
                 </td>
               </tr>
+              <tr class="webgui_managementaccess_choices webgui_managementaccess_ip">
+                <td><a id="help_for_webguiip" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext("Management IP") ?></td>
+                <td>
+                  <input name="webguiip" type="text" value="<?= $pconfig['webguiip'] ?>"/>
+                  <?=gettext("Only accept connections from the specified IP. Use with care."); ?>
```

Comment:
> Only accept connections _from_ the specified IP.
Reads like the allowed remote IPs (i.e. source IP to connect to OPNsense). I recommend rewording this to something like
* _Only accept connections on the specified IP._
* _Management services only listen on specified IP address._